connectors/src/connectors/google_drive/temporal/worker.ts

```diff
@@ -32,6 +33,15 @@ export async function runGoogleWorkers() {
         () => new GoogleDriveCastKnownErrorsInterceptor(),
       ],
     },
+    bundlerOptions: {
```

Comment:
I think we tried this and it breaks prod (but could be wrong)

Reply:
It's suggested by @flvndvd and copy-pasted from the same file for confluence. I think we are fine.

Reply:
Yes it does work @fontanierh, only Confluence so far uses it.

Reply:
cool 2 should probably align all workers at some point